oeps/oep-templates/adr-based-template.rst

```diff
@@ -27,7 +27,7 @@ OEP-XXXX: OEP Template
 +-----------------+--------------------------------------------------------+
 | Created         | <date created on, in YYYY-MM-DD format>                |
 +-----------------+--------------------------------------------------------+
-| `Review Period` | <start - target end dates for review>                  |
+| Review Period   | <start - target end dates for review>                  |
 +-----------------+--------------------------------------------------------+
```

Comment:
@nedbat While you're here, can you update the templates to use the more maintainable formatting for these tables? See the [formatting change made in this PR](https://github.com/edx/open-edx-proposals/pull/128/files#diff-060835d3f05b6411b8ba8d35a0f852c6L5-R23).

Reply:
Done and pushed here.